src/api/nexusaiAPI.js

```diff
@@ -191,13 +191,76 @@ export default {
     read({ projectUuid }) {
       /* return {
         data: {
-          content_base_uuid: '150cf2c6-e8ff-4135-a95d-e0e32aece0ba',
-          intelligence_uuid: '1a40e439-c2c1-4386-ba99-b51df1b53eae',
+          uuid: '150cf2c6-e8ff-4135-a95d-e0e32aece0ba',
+          intelligence: '1a40e439-c2c1-4386-ba99-b51df1b53eae',
         },
       }; */
 
       return request.$http.get(`api/${projectUuid}/router/`);
     },
+
+    actions: {
+      create({ projectUuid, flowUuid, name, description }) {
+        return request.$http.post(`api/${projectUuid}/flows/`, {
+          uuid: flowUuid,
+          name: name,
+          prompt: description,
+          fallback: false,
+        });
+      },
+
+      edit({ projectUuid, actionUuid, description }) {
+        return request.$http.patch(`api/${projectUuid}/flows/${actionUuid}/`, {
+          prompt: description,
+        });
+      },
+
+      delete({ projectUuid, actionUuid }) {
+        return request.$http.delete(`api/${projectUuid}/flows/${actionUuid}/`);
+      },
+
+      list({ projectUuid }) {
+        return request.$http.get(`api/${projectUuid}/flows/`);
+      },
+
+      flows: {
+        list({ next, projectUuid, name }) {
+          if (next) {
+            return request.$http.get(`api/${projectUuid}/search-flows/${next}`);
+          }
+
+          /* return new Promise((resolve) => {
+            delay(resolve, 1000, {
+              data: {
+                count: 3,
+                next: null,
+                previous: null,
+                results: [
+                  {
+                    uuid: '83b3b715-e4db-4740-81eb-feacd4dc9d88',
+                    name: 'M - Switch language',
+                  },
+                  {
+                    uuid: '9e3d5d1f-e5f7-422a-9fce-995ef80996d4',
+                    name: 'M - Direct response - Switch language',
+                  },
+                  {
+                    uuid: '36a8abe8-3c37-451f-9b5d-cc3aef54811d',
+                    name: 'M - Language Question',
+                  },
+                ],
+              },
+            });
+          }); */
```

Comment:
Some commented codes are still there because we are having some troubles with back integrations, but it will be removed as soon as these inconsistencies no longer occur.